src/views/search/search.jsx

```diff
@@ -0,0 +1,148 @@
+var injectIntl = require('react-intl').injectIntl;
+var FormattedMessage = require('react-intl').FormattedMessage;
+var React = require('react');
+var render = require('../../lib/render.jsx');
+
+var Api = require('../../mixins/api.jsx');
+
+var Page = require('../../components/page/www/page.jsx');
+var Box = require('../../components/box/box.jsx');
+var SubNavigation = require('../../components/subnavigation/subnavigation.jsx');
+var Tabs = require('../../components/tabs/tabs.jsx');
+var Carousel = require('../../components/carousel/carousel.jsx');
+var offset = 0;
+var more = [];
+var tab = 'projects';
+var searchTerm = '';
+
+require('./search.scss');
+
+var Search = injectIntl(React.createClass({
+    type: 'Search',
+    mixins: [
+        Api
+    ],
+    getInitialState: function () {
+        return {};
+    },
+    componentDidMount: function () {
```

Comment:
Add a `@todo` comment here to make note of how this should be migrated to React-Router once available.